app/controllers/users_controller.rb

```diff
@@ -1,5 +1,21 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.order(:id)
+  end
 
-  def show; end
+  def show
+    @user = User.find(params[:id])
+    @posts = Post.where(author_id: @user.id)
+  end
+
+  def created
+    @user = User.create(user_params)
+    redirect_to @user
+  end
```

Comment:
- [ ] The function name should be `create` as this is what rails CRUD operation naming convention is. Please update this.

Reply:
Done ✅ .. Thanks for your review 👍🏼